### ISSUE TITLE
Render with Vulkan API?

### ISSUE THREAD

**Issue by danke-sra** [Samsung/GearVRf#422]
Can we look into this feature since Vulkan has been released.

Vulkan 1.0 specification has been released on February 16th, 2016 and Khronos members released Vulkan drivers and SDKs on the same day. From https://www.khronos.org/vulkan/

**Comment by thomasflynn** [comment]
Yep, vulkan is in the plans.  Need to meet some internal deadlines first though.
